src/scripts/main.js

```diff
@@ -1,5 +1,23 @@
 'use strict';
 
-// const pushNotification = (top, right, title, description, type) => {
-//   // write code here
-// };
+const pushNotification = (top, right, title, description, type) => {
+  const main = document.querySelector('body');
+  const message = document.createElement('div');
+  const titleMessage = document.createElement('h2');
+  const descriptionMessage = document.createElement('p');
+
+  message.classList.add('notification', type);
+  message.style.cssText = `top: ${top}px; right: ${right}px`;
+  titleMessage.classList.add('title');
+  titleMessage.innerText = title;
+  descriptionMessage.innerText = description;
+  message.append(titleMessage, descriptionMessage);
+  main.append(message);
+
+  window.setTimeout(function() {
```

Comment:
window писати не обов'язково
```suggestion
  setTimeout(function() {
```